topics/outcomes-metrics.yaml

```diff
@@ -0,0 +1,16 @@
+topic: outcomes-metrics
+pipeline: outcomes
+description: Outcomes for metrics sent to Sentry
+services:
+  producers:
+    - getsentry/relay
+  consumers:
+    - getsentry/snuba
+    - getsentry/super-big-consumers
```

Comment:
Is this pipeline meant to bypass the current metrics outcomes pipeline that takes data in after the indexer ? If yes can we get rid of the old one

Reply:
I am not familiar with any (other) metrics outcomes pipeline, so I assume the answer is yes.

Reply:
This is the billing metrics consumer https://github.com/getsentry/sentry/blob/0730d277c3ad16c4c8aaebd7b2edd51f08819dea/src/sentry/ingest/billing_metrics_consumer.py#L41

I cannot find the design docs anymore.